static/app/views/discover/results.tsx

```diff
@@ -690,6 +692,44 @@ export class Results extends Component<Props, State> {
     return null;
   }
 
+  renderTransactionsDatasetDeprecationBanner() {
+    const {savedQueryDataset} = this.state;
+    const {location, organization} = this.props;
+    const dataset = getDatasetFromLocationOrSavedQueryDataset(
+      location,
+      savedQueryDataset
+    );
+    if (
+      this.state.showTransactionsDeprecationAlert &&
+      organization.features.includes('performance-transaction-deprecation-banner') &&
+      dataset === DiscoverDatasets.TRANSACTIONS
+    ) {
+      return (
+        <Alert.Container>
+          <Alert
+            type="warning"
+            trailingItems={
+              <StyledCloseButton
+                icon={<IconClose size="sm" />}
+                aria-label={t('Close')}
+                onClick={() => {
+                  this.setState({showTransactionsDeprecationAlert: false});
+                }}
+                size="zero"
+                borderless
+              />
+            }
+          >
+            {tctCode(
+              'The transactions dataset is going to be deprecated soon. Please use Explore->Traces with the [code:is_transaction:true] filter instead.'
```

Comment:
I think we can make this a link and use the form `Explore / Traces` with it linkified?